eslint.config.cjs

```diff
@@ -1,154 +1,160 @@
+        ignores: [
+            "res/controllers/lodash.mixxx.js",
+            "res/controllers/Novation-Launchpad MK2-scripts.js",
+            "res/controllers/Novation-Launchpad Mini MK3-scripts.js",
+            "res/controllers/Novation-Launchpad-scripts.js",
+        ],
     },
-
-    settings: {
-        jsdoc: {
-            preferredTypes: {
-                object: "Object",
-            },
+    {
+        languageOptions: {
+            ecmaVersion: 7,
+            sourceType: "script",
         },
     },
-
-    rules: {
-        "array-bracket-spacing": "warn",
-        "block-spacing": "warn",
-
-        "brace-style": ["warn", "1tbs", {
-            allowSingleLine: true,
-        }],
-
-        curly: "warn",
-        camelcase: "warn",
-        "comma-spacing": "warn",
-
-        "computed-property-spacing": ["warn", "never", {
-            enforceForClassMembers: true,
-        }],
-
-        "dot-location": ["warn", "property"],
-        "dot-notation": "warn",
-        eqeqeq: ["error", "always"],
-        "func-call-spacing": "warn",
-
-        "func-style": ["error", "expression", {
-            allowArrowFunctions: true,
-        }],
-
-        indent: ["warn", 4],
-        "key-spacing": "warn",
-        "keyword-spacing": "warn",
-        "linebreak-style": ["warn", "unix"],
-        "newline-per-chained-call": "warn",
-        "no-constructor-return": "warn",
-        "no-extra-bind": "warn",
-        "no-sequences": "warn",
-        "no-useless-call": "warn",
-        "no-useless-return": "warn",
-        "no-trailing-spaces": "warn",
-
-        "no-unneeded-ternary": ["warn", {
-            defaultAssignment: false,
-        }],
-
-        "no-unused-vars": ["error", {
-            argsIgnorePattern: "^_",
-        }],
-
-        "no-var": "warn",
-
-        "object-curly-newline": ["warn", {
-            consistent: true,
-            multiline: true,
-        }],
-
-        "object-curly-spacing": "warn",
-        "prefer-const": "warn",
-        "prefer-regex-literals": "warn",
-        "prefer-template": "warn",
-        quotes: ["warn", "double"],
-        "require-atomic-updates": "error",
-        semi: "warn",
-        "semi-spacing": "warn",
-        "space-before-blocks": ["warn", "always"],
-        "space-before-function-paren": ["warn", "never"],
-        "space-in-parens": "warn",
-        yoda: "warn",
-    },
-}, ...compat.extends("plugin:@typescript-eslint/recommended").map(config => ({
-    ...config,
-    files: ["res/controllers/*.d.ts"],
-})), {
-    files: ["res/controllers/*.d.ts"],
-
-    rules: {
-        "no-unused-vars": "off",
+    {
+        plugins: {
+            jsdoc,
+        }
     },
-}, {
-    files: ["**/*.mjs"],
-
-    languageOptions: {
-        ecmaVersion: 5,
-        sourceType: "module",
+    eslint.configs.recommended,
+    tseslint.configs.recommended,
+    jsdoc.configs['flat/recommended'],
+    diffConfig, // TODO FIXME
```

Comment:
What is this `TODO FIXME`? If it still relevant, would you mind adding more context with the issue? Since this part of the CI process, I don't think there is need for tracking into an issue, but it would be great to keep the problem self explanatory.

Reply:
whoops, nope, irrelevant anymore. I'll fix the commit and force push.

Reply:
done